docs/reference/commandline/ps.md

```diff
@@ -51,6 +51,7 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 * ancestor (`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`) - filters containers that were created from the given image or a descendant.
+* isolation (default|hyperv)
```

Comment:
sorry for spamming (and not even in docs review yet); IIUC this also affects the API, so this needs to be in https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.22.md as well :relaxed: